README.md

```diff
@@ -92,19 +98,20 @@ cpus = 5 # For multiprocessing, usually a good idea to use multiple CPUs as long
 n_trials = 50 #Number of trials to simulate
 
 ##### Here we define the sources of the brain activity (event) for each trial
+# Because the last source determines the response, we will get four events during the trials, and therefore five stages
 n_sources = 5
 frequency = 10.#Frequency of the event defining its duration, half-sine of 10Hz = 50ms
 amplitude = .5e-6 #Amplitude of the event in nAm, defining signal to noise ratio
 shape = 2 #shape of the gamma distribution
 means = np.array([60, 150, 200, 100, 80])/shape #Mean duration of the stages in ms
-names = simulations.available_sources()[:n_sources+1]#Which source to activate at each stage (see atlas when calling simulations.available_sources())
+names = simulations.available_sources()[:n_sources]#Which source to activate at each stage (see atlas when calling simulations.available_sources())
```

Comment:
It should be +1 as the button event/response recording is simulated as being an event in the EEG. Not super clean to do so but fine so far

Reply:
Ah no I got confused and thought n_sources was still 4